parser/parser_test.go

```diff
@@ -97,6 +97,16 @@ func (s *testParserSuite) TestSimple(c *C) {
 		c.Assert(err, IsNil, Commentf("source %s", src))
 	}
 
+	// Testcase for not keyword token
+	notKeywordTokens := []string{
```

Comment:
This test is unnecessary because it's always right, but once you add it, you need to maintain it.....